Cryptopia/Cryptopia.swift

```diff
@@ -102,6 +102,27 @@ public struct Cryptopia {
         }
     }
     
+    public class CryptopiaOrder: Order {
+
+        override init(json: [String: Any]) {
+            super.init(json: json)
+            if let orderId = json["OrderId"] {
+                self.id = String(describing: orderId)
+            }
+            self.rate = NSDecimalNumber(json["Rate"])
+            self.amount = NSDecimalNumber(json["Amount"])
+            self.remaining = NSDecimalNumber(json["Remaining"])
+            self.total = NSDecimalNumber(json["Total"])
+            if let market = json["Market"] as? String {
+                let split = market.components(separatedBy: "/")
```

Comment:
Use convenience init method from CurrencyPair extension in Cryptopia.swift to create CurrencyPair in Order init to avoid redundant code.